### DISCUSSION TITLE
Hacktoberfest participation?

### DISCUSSION THREAD

**Discussion by tstabrawa** [TD22057/insteon-mqtt#451] [General]
Looks like all you need to do as a maintainer is add the 'hacktoberfest' topic to the project.  Merged PR's then count towards the submitter's goal of 4 PR's in October.

More info: https://hacktoberfest.digitalocean.com/

**Comment by krkeegan** [comment]
It looks like if we just add the tag `hacktoberfest` to the repo it would count?  I don't see any downside to this.  Sadly, I do not have the permissions to add tags.

**Comment by TD22057** [comment]
Added the tag.  There are also hacktoberfest2021 suggested tags and I couldn't find any actual instructions for repos anywhere so 🤷

**Reply by krkeegan** [reply]
Cool, that seems to have done it.
